### DISCUSSION TITLE
adguardhome-sync

### DISCUSSION THREAD

**Discussion by sunjam** [MichaIng/DietPi#7334] [Software Request]
[adguardhome-sync](https://github.com/bakito/adguardhome-sync) - synchronize AdGuard Home config to replicas.  It is an excellent pairing with DietPi, since it already supports Adguard and Unbound.  Thank you for considering!

Link to features supported in sync:

    General Settings
    Filters
    Rewrites
    Services
    Clients
    DNS Config
    DHCP Config
    Theme

**Comment by BelgarionNL** [comment]
I would love this. but it hasnt happened in 2 years so I guess not.
